src/software/ai/hl/stp/play/shoot_or_chip_play.cpp

```diff
@@ -149,12 +132,6 @@ void ShootOrChipPlay::getNextTactics(TacticCoroutine::push_type &yield,
         // We want this second in priority only to the goalie
         result.insert(result.begin() + 1, shoot_or_chip_tactic);
 
-        // If we can't do anything else then patrol?
-        for (auto &patrol_tactic : patrol_tactics)
-        {
```

Comment:
Was this causing a bug?

Reply:
It was causing 8 tactics to be assigned so the tactic-assignment code wasn't happy (although it didn't _creash_, just spammed logs)

Reply:
it was meant to fix a bug where up to 2 of the tactics weren't applicable, but that might not be a problem anymore